pom.xml

```diff
@@ -11,32 +11,30 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <java.version.prop>1.8</java.version.prop>
+        <java.version.prop>11</java.version.prop>
```

Comment:
Just a minor nitpick but I would prefer if you would drop the `.prop` suffix it's used no where else and imo adds no value